vtex-apps/b2b-checkout-settings
# [B2BCHCKOUT-24] Fixed PO Number field display in locales without translations

**What problem is this solving?**
- Messages were only available in EN and PT so the PO Number field was not displaying in ES locale. Fixed bug to default to EN messages if locale doesn't have translations.
- Added Spanish translations.
- Fixed bug to set `hasPONumber` to `true` if orderForm has already been configured to accept `purchaseOrderNumber` field. 

**How to test it?**
Test in workspace [annab2bcheckout](https://annab2bcheckout--sandboxusdev.myvtex.com/):

1. Login as a B2B user with access to checkout
2. Switch locale to ES
3. Add item to cart and go to checkout
4. See PO Number field

_Before_
<img width="1440" alt="b2b-checkout-es-before" src="https://user-images.githubusercontent.com/53097865/172212695-8d94ad03-8bf9-4150-925b-b78c85e7d0cf.png">

_After_
<img width="1440" alt="b2b-checkout-es-after" src="https://user-images.githubusercontent.com/53097865/172212713-574c544d-2ed9-4543-b0ef-6f1b0c89dd6f.png">
